packages/facility-server/app/routes/apiv1/medication.js

```diff
@@ -458,6 +458,89 @@ medication.get(
   }),
 );
 
+const givenMarUpdateSchema = z.object({
+  dose: z.object({
+    doseAmount: z.number(),
+    givenTime: z.string().datetime(),
+  }),
+});
+medication.put(
+  '/mar/:id/given',
```

Comment:
is MAR standard or something we've used elsewhere? would prefer the meaning be spelled out if not

Reply:
(in the route and also in exported functions elsewhere; local variables can remain short though something like `const record = await...` might be clearer.)